internal/compiler/passes.rs

```diff
@@ -279,6 +279,7 @@ pub async fn run_passes(
             });
 
             // This is not perfect, as this includes translations that may not be used.
+            #[cfg(feature = "bundle-translations")]
```

Comment:
This change is actually unrelated, it allow to compile the screenshot test standalone